app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -42,6 +42,7 @@ def setup_form_object(step)
           begin
             form_name.constantize
           rescue NameError => x
```

Comment:
Ah! it's in the dummy apps `config/environments/development.rb`:

```
  # Raises error for missing translations
  config.action_view.raise_on_missing_translations = true
```
I'll change it to `false` as part of what I'm working on at the moment, and then drop the generic file.